app/segments/drill/[id]/submission/input.js

```diff
@@ -137,6 +141,136 @@ async function uploadAttempt(
 }
 
 //TODO: Create a function to check leaderboard and update if needed
+function checkLeaderboardUpdate(uploadData, drillInfo, currentLeaderboard) {
```

Comment:
rename this to handleLeaderboardUpdate because checkLeaderboardUpdate implies that no update will happen